.github/workflows/test-components.yml

```diff
@@ -20,65 +20,65 @@ jobs:
       - id: get-versions
         uses: ./.github/actions/get-python-versions
 
-  # Job to define test sessions based on changed files
-  define-test-sessions:
-    runs-on: ubuntu-latest
-    outputs:
-      test-sessions: ${{ steps.test-sessions.outputs.test-sessions }}
-    steps:
-      - uses: actions/checkout@v4
-        with:
-          fetch-depth: 0 # Use a deep clone to get target PR branch
+  # # Job to define test sessions based on changed files
```

Comment:
It might be easier to just temporarily rename the file to `test_foo.yaml.disabled` during the debugging tasks in the PR